resources/glossary.md

```diff
@@ -30,6 +30,9 @@ shell
   
 terminal
   a program that makes shell visible for us and allows for interactions with it
+  
+ directory 
+  a collection of files typically created for organizational purposes
```

Comment:
```suggestion
directory 
  a collection of files typically created for organizational purposes
```

Reply:
the lack of spacing is important to make the glossary work. commit this in to acknowledge and then I'll merge.